### ISSUE TITLE
Emily Quests on Demand (Gold Sink)

### ISSUE THREAD

**Issue by grekan** [prophecymud/ideas#98]
Some sort of triggerable emily quest where you have to fight your way through a maze/arena and you earn <x>, even if you die, because you're quested you can keep going to try and win. You can others can buy in so you can play together. 

Basically pay Emily to fire up a dungeon quest for you, and you have maybe an 30-60 minutes to complete it for a reward. It could be level dependent with the highest level participant setting the mob intensity. 

You'd want it to be expensive enough to justify the reward along with the work it takes, but also have the reward be worth the effort. Perhaps have a 24 or 48 hour timer on it per person so people can't keep retriggering it and abuse it for qps or whatever the reward is. 

Maybe 50k gold to trigger it, and then if you succeed at the quest you get a qp or two or qp and aqpts, and if you lose maybe you get 100aqpts for your time?

Just spit balling on the rewards.

Basically I see it like a Ctuchik quest.
